ridemyway/api/vechicle/models.py

```diff
@@ -0,0 +1,13 @@
+from django.conf import settings
+from django.db import models
+
+
+class Vechicle(models.Model):
+    number_plate = models.CharField(max_length=20, unique=True)
+    category = models.CharField(max_length=50)
+    color = models.CharField(max_length=20, blank=True, null=True)
+    user_id = models.ForeignKey(
+        settings.AUTH_USER_MODEL, related_name="vechicles", on_delete=models.CASCADE
+    )
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
Great work @PatrickCmd, however you . could make use of the base model for some of these attributes. https://github.com/marthamareal/ridemyway-api-django/blob/c70a4cfd56da32035859eb7ca4806667f27ff7a8/ridemyway/api/utilities/base_classes/models.py#L6

Reply:
Done and thanks